website/assets/css/common.css

```diff
@@ -79,6 +79,13 @@ header a {
 header .navlink {
   color: #363434;
 }
+
+header .news {
```

Comment:
I would prefer to not style news differently than the other sections to keep the header uniform. What are your thoughts?

Reply:
I would prefer to have the news a different color. Something outstanding that people will go and check it out among the first.  Otherwise, what's the point of news, if it's not catchy.

Reply:
Sounds good!